src/App.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import NavBar from './components/navbar';
+import BookCategory from './components/bookcategory';
+import DisplayBooks from './components/showbook';
+
+const App = () => (
+  <>
+    <BrowserRouter>
+      <NavBar />
+      <Routes>
+        <Route path="/" element={<DisplayBooks />} />
```

Comment:
- [OPTIONAL] You did a great job initializing your app with components! I would recommend using `<Route exact path="/" ... >` this way you will avoid visitors going into the same path when using `/a` `/b` `/c` or any path that starts with `/`.